subsys/bluetooth/audio/pacs.c

```diff
@@ -54,20 +54,20 @@ struct pacs {
 	sys_slist_t list;
 };
 
-#if defined(CONFIG_BT_PAC_SNK)
-static uint16_t snk_available_contexts;
-static const uint16_t snk_supported_contexts = CONFIG_BT_PACS_SNK_CONTEXT;
+#if defined(CONTIG_BT_PAC_SNK)
+static uint16_t snk_available_contexts = BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED;
+static uint16_t snk_supported_contexts = BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED;
```

Comment:
Since we are removing the use of `CONFIG_BT_PACS_SNK_CONTEXT` here, and adding support for dynamic supported context types, I suggest that we remove the Kconfigs

Reply:
Done